docs/odata-json-format/odata-json-format.md

```diff
@@ -3106,6 +3106,9 @@ reflected in the `Transfer-Encoding` header.
 
 A `body` MUST NOT be specified if the `method` is `get` or `delete`.
 
+The request object and the `headers` object MUST NOT contain name/value pairs with duplicate names.
+This is in conformance with [RFC7493](#rfc7493).
+
```

Comment:
Where did that change come from?

Does this PR depend on some other PR(s) to be merged first?